MiniBootcamp/MiniBootcamp/TweetFeed/TweetCell/TweetCellViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  TweetCellViewModel.swift
+//  MiniBootcamp
+//
+//  Created by Edgar Solis on 27/05/23.
+//
+
+import UIKit
+
+struct TweetCellViewModel {
+    let userName: String
+    let profileName: String
+    let profilePictureName: String
+    let content: String
+    
+    var profilePicture: UIImage? {
+        guard let image = UIImage(named: profilePictureName) else { return UIImage(named: "cat") }
+        return image
```

Comment:
After a guard let statement, you should leave and empty line. Also, if you are going to write something on the else statement, try to use a new line. You can leave it in just one when you are using only a return.
Example:

```suggestion
        guard let image = UIImage(named: profilePictureName) else { 
              return Constant.catImage 
        }
        
        return image
```